src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -203,6 +218,40 @@ sample({
   target: $accounts,
 });
 
+sample({
+  clock: walletRemoved,
+  source: $accounts,
+  fn: (accounts, wallet) => ({ accounts: accounts.filter((a) => a.walletId === wallet.id), wallet }),
```

Comment:
Time for utility function has come 😄
like:
```javascript
function getWalletAccounts(wallet: Wallet, accounts: Account[]): Account[]
```
maybe even generic one to get related accounts
like:
```javascript
function getWalletAccounts<T extends Account>(wallet: Wallet, accounts: T[]): T[]
```